scripts/test.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-python -m unittest discover "tests/unit"
+find tests/unit -type d | grep -v __pycache__ | xargs -n 1 python -m unittest discover
```

Comment:
i'm getting an error running `docker-compose run bot ./scripts/test.sh` because some of the requirements aren't installed. this fixed my issue

```suggestion
grep -v -e pydocstyle -e yapf -e pre-commit requirements-dev.txt | xargs pip install
find tests/unit -type d | grep -v __pycache__ | xargs -n 1 python -m unittest discover
```

Reply:
i just copied this from `integration-test.sh`